server/services/yeelight/lib/models/index.js

```diff
@@ -0,0 +1,9 @@
+const color = require('./color');
+const white = require('./white');
+const unhandled = require('./unhandled');
+
+module.exports = {
+  color,
+  white,
+  unhandled,
```

Comment:
Do you manage this kind of "no feature device" in front? As we can't create device without features.

Reply:
Yes, in the `YeelightDeviceBox.jsx` file, line 64.